### PR TITLE
improve tf.nn.l2_normalize() doc

### PR DESCRIPTION
**Pull request by stmugisha**
Added usage example to improve documentation.

### CODE REVIEW

**Review by alextp — CHANGES_REQUESTED** [review]
Can you write these in doctest format, with >>> etc?

File: tensorflow/python/ops/math_ops.py [hunk 1]
```diff
@@ -317,7 +317,22 @@ def __div__(self, y):
 @tf_export("math.divide", "divide")
 @dispatch.add_dispatch_support
 def divide(x, y, name=None):
-  """Computes Python style division of `x` by `y`."""
+  """Computes Python style division of `x` by `y`.
+  
+  For example:
+  ```python
```
Comment (yashk2810) [inline tensorflow/python/ops/math_ops.py]: You don't need the ```python backticks anymore.

Comment (yashk2810) [reply tensorflow/python/ops/math_ops.py]: Also please keep a newline between the doctest and the markdown text.

```
For example

>>>
>>>

Args
```

File: tensorflow/python/ops/math_ops.py [hunk 2]
```diff
@@ -3290,29 +3305,46 @@ def cumsum(x, axis=0, exclusive=False, reverse=False, name=None):
...
   ```
 
   By setting the `exclusive` kwarg to `True`, an exclusive cumsum is performed
   instead:
 
   ```python
-  tf.cumsum([a, b, c], exclusive=True)  # [0, a, a + b]
+  # tf.cumsum([a, b, c], exclusive=True)  => [0, a, a + b]
+  >>> x = tf.constant([2, 4, 6, 8])
+  >>> tf.cumsum(x, exclusive=True)  # [ 0,  2,  6, 12]
```
Comment (yashk2810) [inline tensorflow/python/ops/math_ops.py]: The output should be below.

```
>>> tf.cumsum(x, exclusive=True)
Output goes here
```

Please see: https://www.tensorflow.org/community/contribute/docs_ref#docstring_considerations

File: tensorflow/python/ops/nn_impl.py [hunk 3]
```diff
@@ -626,6 +626,18 @@ def l2_normalize_v2(x, axis=None, epsilon=1e-12, name=None):
 
   For `x` with more dimensions, independently normalizes each 1-D slice along
   dimension `axis`.
+  For example:
+  
+  ```python
+  import tensorflow as tf
```
Comment (yashk2810) [inline tensorflow/python/ops/nn_impl.py]: Import is not needed.

Comment (steph-en-m) [reply tensorflow/python/ops/nn_impl.py]: Alright, I had to first cleanup my commit history then work on the changes now.

**Pull request closed without merging**